js_modules/dagit/packages/core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -224,7 +220,10 @@ export const useMaterializationAction = (preferredJobName?: string) => {
     }
     setState({type: 'loading'});
 
-    const result = await client.query<LaunchAssetLoaderQuery, LaunchAssetLoaderQueryVariables>({
+    const result = await client.query<
+      LaunchAssetLoaderQueryQuery,
+      LaunchAssetLoaderQueryQueryVariables
+    >({
```

Comment:
Hmm I wonder if these can be auto-inferred by typescript now that the types are bound in some way to `LAUNCH_ASSET_LOADER_QUERY`? (I'm sure you tried and that's not the case but I wish 😅)

Reply:
I have actually not tried it for these `client.query` calls, but it would be nice.